go/vt/servenv/grpc_server.go

```diff
@@ -83,6 +83,10 @@ var (
 	// The lower bound for window size is 64K and any value smaller than that will be ignored.
 	GRPCInitialWindowSize = flag.Int("grpc_server_initial_window_size", 0, "grpc server initial window size")
 
+	// EnforcementPolicy MinTime that sets the keepalive enforcement policy on the server.
+	// This is the minimum amount of time a client should wait before sending a keepalive ping.
+	GRPCEnforcementPolicyMinTime = flag.Duration("grpc_enforcement_min_time", 5*time.Minute, "grpc server minimum keepalive time")
```

Comment:
Actually I clicked approve a bit too soon :)

IMO we should be clearer that this has to do with keepalives, so I would change the flag name to `grpc_keepalive_enforcement_policy_min_time` and the variable name to the corresponding camelCase.